onbuild/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#
+# Dockerfile for Storm Mesos framework
+#
+FROM mesosphere/mesos:0.27.0-0.2.190.ubuntu1404
```

Comment:
@DarinJ Awesome! Exactly what we have been wanting!

@erikdw @lfundaro @DarinJ With @DarinJ's finding, we probably dont even need a separate folder for versioning - do we? If so, I am slightly concerned about having a folder for mesos version but not for storm version. Thoughts?